floodviz/static/js/peakflow.js

```diff
@@ -7,7 +7,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
 	peakflow_bar.style.height = FV.peakmeta['height'];
 	peakflow_bar.style.width = FV.peakmeta['width'];
 
-	var margin = {bottom: 100, right: 120, left: 120, top: 150};
+	var margin = {bottom: 40, right: 40, left: 40, top: 75};
```

Comment:
This isn't a problem with this PR, but these should probably not be set in javascript.